auction-app-frontend/src/pages/Home/Home.jsx

```diff
@@ -1,7 +1,134 @@
-import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+import {
+  ProductGrid,
+  ErrorComponent,
+  LoadingComponent,
+  Button,
+} from "src/components";
+
+import * as productService from "src/services/ProductService";
+import * as categoryService from "src/services/CategoryService";
+
+import { go } from "src/assets/icons";
+
+import "./style.scss";
 
 const Home = () => {
-  return <div>Home Page</div>;
+  const [activeTab, setActiveTab] = useState("newArrivals");
```

Comment:
Maybe we can create constants for newArrivals and lastChance and use them instead of raw strings